video/index.js

```diff
@@ -13,6 +18,12 @@ class IrnmnVideo extends HTMLElement {
         this.setupButton();
     }
 
+    attributeChangedCallback(name, oldValue, newValue) {
+        if (name === 'src' && oldValue !== newValue) {
```

Comment:
Thanks @samuelsilvapt I've added a comment to explain the need for this callback. I kept the 'src' conditional check just in case we add more attributes in the future